thread-manager/examples/core_contention_basics.rs

```diff
@@ -48,11 +48,12 @@ fn main() -> anyhow::Result<()> {
         println!("Running {exp}");
         let mut conffile = PathBuf::from(env!("CARGO_MANIFEST_DIR"));
         conffile.push(exp);
-        let conffile = std::fs::File::open(conffile)?;
```

Comment:
Typically, we don't specify configurations in files. Doing it in code instead for the example/test cases, or with command line flags for other cases. Sometimes use yml (for accounts information for example).

Reply:
I got you, you want to read `thread-manager/examples/core_contention_contending_set.toml` with it. 
Yeah, could you maybe have a method that reads this file and used in both example and production environment in the future?

Reply:
this file will never be read in prod, its just for examples which are also benchmarks/tests. I'm not even sure we will use toml in prod for this yet, though it is likely.